src/lightning/pytorch/utilities/types.py

```diff
@@ -82,3 +82,11 @@ class LRSchedulerConfig:
     monitor: Optional[str] = None
     # enforce that the monitor exists for ReduceLROnPlateau
     strict: bool = True
+
+
+class _SizedIterable(Protocol):
+    def __len__(self) -> int:
+        pass
```

Comment:
Shall be here 0 or how the pass works here?

Reply:
I guess mypy knows for types that subclass from Protocol only the signature matters and the implementation is irrelevant.